#Header
imc_ros_interface/Header header 
# Latitude (WGS-84).
std_msgs/Float64 lat
# Longitude (WGS-84).
std_msgs/Float64 lon
# Height (WGS-84).
std_msgs/Float32 height
# Offset north.
std_msgs/Float32 x
# Offset east.
std_msgs/Float32 y
# Offset down.
std_msgs/Float32 z
# Rotation over x axis.
std_msgs/Float32 phi
# Rotation over y axis.
std_msgs/Float32 theta
# Rotation over z axis.
std_msgs/Float32 psi
# Body-Fixed xx Velocity.
std_msgs/Float32 u
# Body-Fixed yy Velocity.
std_msgs/Float32 v
# Body-Fixed zz Velocity.
std_msgs/Float32 w
# Ground Velocity X (North).
std_msgs/Float32 vx
# Ground Velocity Y (East).
std_msgs/Float32 vy
# Ground Velocity Z (Down).
std_msgs/Float32 vz
# Angular Velocity in x.
std_msgs/Float32 p
# Angular Velocity in y.
std_msgs/Float32 q
# Angular Velocity in z.
std_msgs/Float32 r
# Depth.
std_msgs/Float32 depth
# Altitude.
std_msgs/Float32 alt
